frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx

```diff
@@ -82,9 +84,24 @@ function MainNavbarView({
     }
   }, [handleCloseNavbar]);
 
+  const isHomePage = window.location.pathname === "/";
```

Comment:
We can remove this now

Reply:
Did we turn of warnings about unused declarations? For a while now I suppose 🤔

Reply:
I guess it stopped working for typescript files for some reason 🤔